internal/runner/rspec_test.go

```diff
@@ -152,3 +153,34 @@ func TestRspecDiscoveryPattern_ExcludePattern(t *testing.T) {
 		t.Errorf("Rspec.discoveryPattern() diff (-got +want):\n%s", diff)
 	}
 }
+
+func TestRspecGetExamples(t *testing.T) {
+	rspec := NewRspec("./test/mock_rspec_dry_run")
+	files := []string{"spec/spells/expelliarmus_spec.rb"}
+	got, err := rspec.GetExamples(files)
+
+	want := []plan.TestCase{
+		{
+			Format:     plan.TestCaseFormatExample,
+			Identifier: "./spec/spells/expelliarmus_spec.rb[1:1]",
+			Name:       "disarms the opponent",
+			Path:       "./spec/spells/expelliarmus_spec.rb:3",
```

Comment:
Do we want to include a leading `./` to the file path? FYI, when the client discovers the test files using file globbing, it doesn't include the leading `./`.